lib/linked_in/api.rb

```diff
@@ -12,16 +12,16 @@ def initialize(access_token=nil)
       # Protocol version 1.0.0
       @connection = LinkedIn::Connection.new params: default_params, headers: default_headers do |conn|
         conn.request :multipart
-        conn.adapter Faraday.default_adapter
       end
+      @connection.adapter Faraday.default_adapter
```

Comment:
does this blow up when inside the block?

Reply:
No, stupid warning that I tried to fix here https://github.com/LiveWorld/linkedin-v2/pull/1 , but it got reintroduced